core/unit_test/tools/TestAllCalls.cpp

```diff
@@ -85,6 +85,7 @@ int main(int argc, char** argv) {
     Kokkos::Profiling::stopSection(sectionId);
     Kokkos::Profiling::destroyProfileSection(sectionId);
     Kokkos::Profiling::markEvent("profiling_event");
+    Kokkos::Tools::declareMetadata("dogs", "good");
```

Comment:
This is incorrect. It should be:

```
Kokkos::Tools::declareMetadata("dogs", "very-good");
```

Reply:
TRUE. THE GOODNESS OF DOGS SHALL NOT BE UNDEREMPHASIZED